core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -368,7 +389,10 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
         $('.cke_combo_button')
           .css('display', 'none');
       }
-
+      if (!this.internetConnectivityService.isOnline()) {
+        this.connectedToInternet = false;
+        this.disableRTEicons();
+      }
```

Comment:
Why do we need to this check here and setting these values?

Reply:
This check is run when CKEditor is initialized. If we'll not check this then it will always take default values.